src/actions/ast/setInScopeLines.js

```diff
@@ -9,10 +9,10 @@ import { getSourceLineCount } from "../../utils/source";
 
 import { range, flatMap, uniq, without } from "lodash";
 
-import type { ASTLocation } from "../../types";
+import type { AstLocation } from "../../workers/parser";
 import type { ThunkArgs } from "../types";
 
-function getOutOfScopeLines(outOfScopeLocations: ASTLocation[]) {
+function getOutOfScopeLines(outOfScopeLocations: ?(AstLocation[])) {
```

Comment:
Why the `?` change here?

Reply:
`getOutOfScopeLines` has a check if `ASTLocation` is falsey